contracts/Token.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract Token is ERC20 {
```

Comment:
If this is only to be used in testing, then I'd put it elsewhere to indicate that this is not a core contract of the project.

E.g. in `contracts/test` or `contracts/mocks`. I'd also recommend to rename it to "SampleToken" or "TokenMock".